src/node.cc

```diff
@@ -2914,9 +2914,6 @@ void SetupProcessObject(Environment* env,
   env->SetMethod(process, "_setupNextTick", SetupNextTick);
   env->SetMethod(process, "_setupPromises", SetupPromises);
   env->SetMethod(process, "_setupDomainUse", SetupDomainUse);
-
-  // pre-set _events object for faster emit checks
-  process->Set(env->events_string(), Object::New(env->isolate()));
```

Comment:
I see the [`NativeWeakMap`](http://brendanashworth.github.io/v8-docs/classv8_1_1_native_weak_map.html) but I can't find the regular Map either.